example/admin@test.priv

```diff
@@ -1 +1 @@
-881e179b8d6900f892b43c7010c77e47731711735dae462ba9a8d12a9d956e58ab5bce5597a9914b0a743fccd3c56e521c1b59d73391d8ae33cc3667c5068539
\ No newline at end of file
+1d7e0a32ee0affeb4d22acd73c2c6fb6bd58e266c8c2ce4fa0ffe3dd6a253ffb
```

Comment:
it is better not to store autogenerated files in git, but provide an example "how to generate them"

Reply:
May be, but not in our dark reality. Generating all these files (especially genesis block) is a real pain. I tried to do it by myself yesterday night and got some garbage instead a proper genesis block and only @neewy saved me.
Let's save all these stuff here until we have an user-friendly cli.

Reply:
yes!